### PR TITLE
Completed tasks

### PR DESCRIPTION
**Pull request by F-u-n-H-o-u-s-e**
- [DEMO LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Comment by F-u-n-H-o-u-s-e** [comment]
- [DEMO LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,58 @@
...
+      <img 
+      src="images/imac.jpeg" 
+      class="card__img"
+      alt="iMac" 
+      >
+
+      <a 
+      href="#" 
+      class="card__link"
+      >APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
```
Comment (liliia-livandovskaya) [inline src/index.html:23]:
```suggestion
      <a 
        href="#" 
        class="card__link"
      >
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </a>
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,58 @@
...
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <img 
+      src="images/imac.jpeg" 
+      class="card__img"
+      alt="iMac" 
+      >
```
Comment (liliia-livandovskaya) [inline src/index.html:18]:
```suggestion
      <img 
        src="images/imac.jpeg" 
        class="card__img"
        alt="iMac" 
      >
```
fix it all over your code

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  src: url("./font/Roboto-Bold.ttf") format("truetype");
+  font-weight: 700;
+  font-style: bold;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
```
Comment (liliia-livandovskaya) [inline src/style.css:27]:
```suggestion
* {
  box-sizing: border-box;
}
```
in the `*` we can use only `box-sizing`

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+body {
+  font-family: Roboto, serif;
+}
```
Comment (liliia-livandovskaya) [inline src/style.css:31]: its better to set fonts in `html` selector

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+  margin-right: 4px;
+}
+
+div.stars--1 div:nth-child(-n+1) ,
+div.stars--2 div:nth-child(-n+2) ,
+div.stars--3 div:nth-child(-n+3) ,
+div.stars--4 div:nth-child(-n+4) ,
+div.stars--5 div:nth-child(-n+5) {
+  background-image: url("images/star-active.svg");
+}
```
Comment (liliia-livandovskaya) [inline src/style.css:87]: move it to the separate file

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+  color: #616070;
+  margin: 0 0 3px 16px;
+}
+
+.price-box__price {
+  margin-top: 24px;
+}
+
+.price__cost {
+  font-family: Roboto-Bold, serif;
```
Comment (liliia-livandovskaya) [inline src/style.css:123]: move the font-family to the `html` selector and use inherit if needed

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+  text-transform: uppercase;
+}
+
+.card__button {
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  border: #00acdc;
+  font-family: Roboto-Bold, sans-serif;
```
Comment (liliia-livandovskaya) [inline src/style.css:135]: the same there

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.price__text {
+  font-family: Roboto-Normal, sans-serif;
```
Comment (liliia-livandovskaya) [inline src/style.css:112]: move the `font-family` to the `html` selector and use inherit if needed

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
GJ! 👍  pay attention to the comments

File: src/styles/main.css [hunk 9]
```diff
@@ -0,0 +1,129 @@
+
+/* Connect all blocks*/
+@import  "./stars.css";
+
+html {
+  font-family: Roboto, serif;
+}
```
Comment (valerii-derkach) [inline src/styles/main.css:7]: перенеси після імпорту

File: src/styles/main.css [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+
+* {
+  box-sizing: border-box;
+}
+
+/*Block card*/
+.card {
+  width: 200px;
+  height: 408px;
```
Comment (valerii-derkach) [inline src/styles/main.css:38]: пункт 8 [чекліста](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

**Review by valerii-derkach — CHANGES_REQUESTED** [review]

**Pull request closed without merging**